aip_x2_launch/launch/object_recognition_camera.launch.py

```diff
@@ -0,0 +1,185 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.actions import OpaqueFunction
+from launch.conditions import LaunchConfigurationEquals
+from launch.conditions import LaunchConfigurationNotEquals
+from launch.substitutions import EnvironmentVariable
+from launch.substitutions import LaunchConfiguration
+from launch_ros.actions import ComposableNodeContainer
+from launch_ros.actions import LoadComposableNodes
+from launch_ros.descriptions import ComposableNode
+from launch_ros.substitutions import FindPackageShare
+import yaml
+
+
+def launch_setup(context, *args, **kwargs):
+    # https://github.com/ros2/launch_ros/issues/156
+    def load_composable_node_param(param_path):
+        with open(LaunchConfiguration(param_path).perform(context), "r") as f:
+            return yaml.safe_load(f)["/**"]["ros__parameters"]
+
+    composable_nodes = [
+        ComposableNode(
+            package="gscam",
+            plugin="gscam::GSCam",
+            name=["gscam", LaunchConfiguration("camera_id")],
+            namespace="/sensing/camera/",
+            remappings=[
+                (
+                    "camera/image_raw",
+                    [
+                        "camera",
+                        LaunchConfiguration("camera_id"),
+                        "/",
+                        LaunchConfiguration("image_topic"),
+                    ],
+                ),
+                (
+                    "camera/image_raw/compressed",
+                    [
+                        "camera",
+                        LaunchConfiguration("camera_id"),
+                        "/",
+                        LaunchConfiguration("image_topic"),
+                        "/compressed",
+                    ],
+                ),
+                (
+                    "camera/image_raw/compressedDepth",
+                    [
+                        "camera",
+                        LaunchConfiguration("camera_id"),
+                        "/",
+                        LaunchConfiguration("image_topic"),
+                        "/compressedDepth",
+                    ],
+                ),
+                (
+                    "camera/image_raw/theora",
+                    [
+                        "camera",
+                        LaunchConfiguration("camera_id"),
+                        "/",
+                        LaunchConfiguration("image_topic"),
+                        "/theora",
+                    ],
+                ),
+                (
+                    "camera/camera_info",
+                    ["camera", LaunchConfiguration("camera_id"), "/camera_info"],
+                ),
+            ],
+            parameters=[
+                load_composable_node_param("gscam_param_path"),
+                {
+                    "camera.image_raw.jpeg_quality": 80,
```

Comment:
[IMO] How about this option move to `gscam_param_path` ?
This parameter is variable depended on system configuration.

Reply:
addressed at https://github.com/tier4/aip_launcher/pull/33/commits/553f1c2a6a20ea72244c060dd4663e3d2da401f3
https://github.com/tier4/autoware_individual_params.gsm8/pull/12
https://github.com/tier4/autoware_individual_params.jpntaxi/pull/37